datahub-web-react/src/graphql/search.graphql

```diff
@@ -720,8 +721,421 @@ fragment searchResultFields on Entity {
     ... on DataPlatform {
         ...nonConflictingPlatformFields
     }
-    ... on DataProduct {
-        ...dataProductSearchFields
+    ... on Join {
+        urn
+        type
+        joinId
+        properties {
+            ...joinPropertiesFields
+        }
+        editableProperties {
+            ...joinEditablePropertiesFields
+        }
+        ownership {
+            ...ownershipFields
+        }
+        tags {
+            ...globalTagsFields
+        }
+        glossaryTerms {
+            ...glossaryTerms
+        }
+        browsePaths {
+            path
+        }
+    }
+}
+
+
+
+fragment searchResultFieldsWithRelationships on Entity {
+    urn
+    type
+    ... on Dataset {
+        name
+        origin
+        uri
+        platform {
+            ...platformFields
+        }
+        dataPlatformInstance {
+            ...dataPlatformInstanceFields
+        }
+        editableProperties {
+            name
```

Comment:
why are we adding all of this stuff in our graphql files?

Reply:
Removed entries which are not required.